Dockerfile

```diff
@@ -35,15 +42,44 @@ RUN wget https://download.docker.com/linux/debian/gpg \
     && apt-get -y install docker-ce \
     && rm -rf /var/lib/apt/lists/*
 
+# create app user
+RUN groupadd -g $APP_USER_GID $APP_USER_NAME \
+    && useradd -m -u $APP_USER_UID -g $APP_USER_GID $APP_USER_NAME \
+    && usermod -a -G docker $APP_USER_NAME \
+    && printf "%s ALL=(ALL:ALL) NOPASSWD: ALL" "$APP_USER_NAME" >> /etc/sudoers.d/$APP_USER_NAME
+
 COPY ./docker/php.ini /usr/local/etc/php/php.ini
 
 RUN docker-php-ext-install zip \
-	&& curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin/ --filename=composer
+    && curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin/ --filename=composer
 
 COPY composer.* symfony.lock ./
 RUN composer install $COMPOSER_FLAGS --no-scripts --no-autoloader
 
 COPY . .
-RUN composer install $COMPOSER_FLAGS
+RUN composer install $COMPOSER_FLAGS \

```

Comment:
Nechces to uz komplet sjednotis s notification service, kdyz uz se do toho hrabe? :) Vyhodit ten `composer install` z `base` a udelat tam `app` target

Reply:
No moc nechcu, protože to znamená předělat taky docker-composer a přenastavit pipeline

Reply:
Dobre no, jsem to zkusil :)

Reply:
Já to taky zkusil, už jsem to i začal měnít a po čtvrthodině jsem ze zaječí nory zbaběle vycouval